pkg/oci/driver/driver.go

```diff
@@ -94,15 +109,70 @@ func deriveVolumeOCID(regionKey string, volumeName string) string {
 	return volumeOCID
 }
 
+func constructKubeClient() (*kubernetes.Clientset, error) {
```

Comment:
We need to be able to construct the client from the env for the managed use case. Let's discuss offline on Monday.